File: ProjectCRUD.UI/src/app/models/employee.update.dto.ts
import { formatDate } from "@angular/common";
import { Guid } from "guid-typescript";

export class EmployeeUpdateDTO {
    id?: Guid;
    firstName = "";
    lastName = "";
    doB = formatDate(new Date(), 'yyyy-MM-dd', 'en-US');
    email = "";
    skillLevel?: number;
    active = true;
  }  

File: ProjectCRUD/DTOs/EmployeeUpdateDTO.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace ProjectCRUD.DTOs
{
    public class EmployeeUpdateDTO
    {
        public int Id { get; set; }
        public string FirstName { get; set; } = "Name";
        public string LastName { get; set; } = "Surname";
        [DataType(DataType.Date)]
        [Column(TypeName="Date")]
        public DateTime DoB { get; set; }
        public string Email { get; set; } = "Email";
        public int SkillLevel { get; set; } = 1;
        public bool Active { get; set; }
    }
}